### PR TITLE
Document how to contribute a new package to Host OS

### PR DESCRIPTION
**Pull request by lisibann**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory:
+
+1. Create a metadata file named <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is described at https://github.com/open-power-host-os/versions/package_metadata_yaml_template.yaml. You can use this template to start your own metadata file.
+2. Create two subdirectories named <distro_name> and <distro_name>/<distro_version>, e.g. CentOS and CentOS/7.
+3. Create rpm package specification file in <package_name>/<distro_name>/<distro_version>/<package_name>.spec.
+4. Copy the files required to build the package to <package_name>/<distro_name>/<distro_version>/SOURCES/.
+
+If the rpm specification file has a %check macro for running tests during rpm package build, add the following code to allow skipping these tests:
```
Comment (alanoe) [inline CONTRIBUTING.rst]: Isn't this a step 5 ?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+How to contribute a new package to Host OS
+------------------------------------------
+
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory:
+
+1. Create a metadata file named <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is described at https://github.com/open-power-host-os/versions/package_metadata_yaml_template.yaml. You can use this template to start your own metadata file.
+2. Create two subdirectories named <distro_name> and <distro_name>/<distro_version>, e.g. CentOS and CentOS/7.
+3. Create rpm package specification file in <package_name>/<distro_name>/<distro_version>/<package_name>.spec.
```
Comment (alanoe) [inline CONTRIBUTING.rst]: Create => Create a

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+   %bcond_with tests
+
+
+2. Add at the point of %check macro:
+
+::
+
+   %if %{with tests}
+   %check
+   make V=1 check  (or any other code used to verify)
```
Comment (alanoe) [inline CONTRIBUTING.rst]: these parentheses will break code if the block is copied and pasted. They should either be commented or removed. IMHO they could be removed

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+------------------------------------------
+
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory:
+
+1. Create a metadata file named <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is described at https://github.com/open-power-host-os/versions/package_metadata_yaml_template.yaml. You can use this template to start your own metadata file.
+2. Create two subdirectories named <distro_name> and <distro_name>/<distro_version>, e.g. CentOS and CentOS/7.
+3. Create rpm package specification file in <package_name>/<distro_name>/<distro_version>/<package_name>.spec.
+4. Copy the files required to build the package to <package_name>/<distro_name>/<distro_version>/SOURCES/.
```
Comment (alanoe) [inline CONTRIBUTING.rst]: It'd be good to mention that these "files" above refer to the extra files which are not available in one of the sources of the package for some reason.

File: CONTRIBUTING.rst (hunk 3 above)
Comment (lisibann) [inline CONTRIBUTING.rst]: Modified to use comment.

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+How to contribute a new package to Host OS
+------------------------------------------
+
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory:
+
+1. Create a metadata file named <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is described at https://github.com/open-power-host-os/versions/package_metadata_yaml_template.yaml. You can use this template to start your own metadata file.
+2. Create two subdirectories named <distro_name> and <distro_name>/<distro_version>, e.g. CentOS and CentOS/7.
```
Comment (olavphibm) [inline CONTRIBUTING.rst]: I think it' s simpler and less confusing to say:
Create a nested directory named <distro_name>/<distro_version>, e.g. CentOS/7.

File: CONTRIBUTING.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+3. Create a rpm package specification file in <package_name>/<distro_name>/<distro_version>/<package_name>.spec.
```
Comment (olavphibm) [inline CONTRIBUTING.rst]: Very minor: RPM starts with a vowel sound, so it should be "an rpm".

Comment (lisibann) [inline CONTRIBUTING.rst]: done

File: CONTRIBUTING.rst (hunk 5 above)
Comment (lisibann) [inline CONTRIBUTING.rst]: done

**Pull request merged**